vpn.tf

```diff
@@ -77,6 +77,18 @@ resource "azurerm_network_security_group" "main" {
     destination_address_prefix = azurerm_network_interface.main.private_ip_address # "*"?
   }
 
+  security_rule {
+    name                       = "allowed-https-ips-inbound"
+    priority                   = 101
+    direction                  = "Inbound"
+    access                     = "Allow"
+    protocol                   = "Tcp"
+    source_port_range          = "*"
+    destination_port_range     = "443"
+    source_address_prefixes    = values(local.vpn.ssh_allowed_inbound_ips)
```

Comment:
Converted to draft as either we allow every IP, or either we allow only ours but iun that case the local var name should be renamed.

Reply:
We have to allow all IPs for 443 (like for the current VPN machine). Use case: anyone who get the openvpn ocnfiguration (that would be hard to track all IPs for all contributors)

We have to keep the IP restriction for SSH (22) though: it's an admin feature only.